benchmarks/benchmark_factory.py

```diff
@@ -51,6 +57,11 @@ def benchmark_factory(params):
         f"benchmark_name = {name} not supported, choose from:\n{supported_benchmarks()}"
     if name.startswith('nasbench201_'):
         dataset_name = name[len('nasbench201_'):]
+        if dataset_name == 'cifar10-valid':
```

Comment:
Can we just delete this?

Reply:
I wanted to try and be backward compatible.

Reply:
Agree with Aaron that it may be easier to remove this case, this complicates our setup forever and I do not think we loose much by changing the name of one dataset.

Reply:
OK, then I delete it.